### PR TITLE
Change model loading and indexing to use a name

### LINKED ISSUE #10: Access game manager model list using named models
Give a model a name so you can index the array based on its name (maybe provide a list of names that is populated when you add a model to the manager)

### PR DESCRIPTION
**Pull request by TheIceCreamBear**
Makes `manager.modelList` accessible by a common model name rather than an arbitrary index. Fixes #10.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -36,18 +36,21 @@ const main = async () => {
...
+    { model: require('./models/raymanModel.obj'), name: 'rayman' }, 
+    { model: require('./models/cow.obj'), name: 'cow' }
+  ];
 
   await manager.addModels(modelRefs);
 
-  const myRayman = new GameObject(manager.modelList[0], new Physics());
-  const myCow = new GameObject(manager.modelList[1], new Physics());
+  const myRayman = new GameObject(manager.modelList['rayman'], new Physics());
+  const myCow = new GameObject(manager.modelList['cow'], new Physics());
```
Comment (ctcuff) [inline src/index.js:48]: Small nit: you can do `const myRayman = new GameObject(manager.modelList.rayman, new Physics());` instead. Same thing for the cow model

**Review by ctcuff — APPROVED** [review]
LGTM, few small comments.

File: src/index.js [hunk 2]
```diff
@@ -36,18 +36,21 @@ const main = async () => {
...
 
   manager.addObjects([myRayman, myCow]);
 
   myCow.physics.angularVelocity = new Vector3(10, 10, 10);
 
-  const raymanModelExtents = manager.modelList[0].getModelExtent();
+  const raymanModelExtents = manager.modelList['rayman'].getModelExtent();
```
Comment (Armintron) [inline src/index.js:53]: Same thing @ctcuff said can be applied here.

**Review by Armintron — APPROVED** [review]
Other than an extension of @ctcuff's comment, looks good!

**Review by Armintron — APPROVED** [review]

**Pull request merged**